addons/account/views/account_invoice_view.xml

```diff
@@ -431,6 +431,9 @@
                             <field name="date_invoice"/>
                             <field name="date_due" attrs="{'readonly': ['|',('payment_term_id','!=',False), ('state', 'in', ['open', 'paid'])]}" force_save="1"/>
                             <field name="move_name" invisible="1"/>
+                            <field name="reference_type" nolabel="1"  attrs="{'readonly':[('state','!=','draft')]}"/>
+                            <field name="reference" nolabel="1"  attrs="{'readonly':[('state','!=','draft')]}"/>
+                            <field name="reference"/>
```

Comment:
Why displaying twice the reference field? Once in readonly if state is draft and the other always editable. Seems strange to me